spec/features/versions_spec.rb

```diff
@@ -270,6 +270,20 @@
     expect(page).to have_text("Someone (probably via the console) deleted commercial in event #{event.title} in conference #{conference.short_title}")
   end
 
+  scenario 'display changes in event commercials in event history', feature: true, versioning: true, js: true do
+    event_without_commercial = create(:event, program: conference.program)
+    event_with_commercial = create(:event, program: conference.program)
```

Comment:
event_with_comercial is created several times in this file. It would be nice if we move it to a `let` and reuse it.